### PR TITLE
allow product updates to be featured

### PR DESCRIPTION
**Pull request by Pomax**
Closes https://github.com/mozilla/foundation.mozilla.org/issues/4560

### CODE REVIEW

File: network-api/networkapi/buyersguide/pagemodels/product_update.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+class Update(models.Model):
+    source = models.URLField(
+        max_length=2048,
+        help_text='Link to source',
+        blank=True,
+    )
+
+    title = models.CharField(
+        max_length=256,
+        blank=True,
```
Comment (mmmavis) [inline network-api/networkapi/buyersguide/pagemodels/product_update.py:13]: This model doesn't have any required fields. Can we make `title` a required field?

File: network-api/networkapi/buyersguide/pagemodels/product_update.py [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    )
```
Comment (mmmavis) [inline network-api/networkapi/buyersguide/pagemodels/product_update.py:14]: This model doesn't have any required fields. Can we make `title` a required field?

**Comment by mmmavis** [comment]
Would be awesome if we could add a column for `featured` on Updates' admin panel so at a glance it's easy to spot featured updates.

**Comment by Pomax** [comment]
good calls, added in!

**Comment by Pomax** [comment]
filed https://github.com/mozilla/foundation.mozilla.org/issues/4565 to investigate that seed value oddity

**Pull request merged**